packages/valory/skills/ceramic_read_abci/tests/test_behaviours.py

```diff
@@ -322,6 +342,9 @@ def test_raises_b(self) -> None:
         params = cast(SharedState, self._skill.skill_context.params)
         params.__dict__["_frozen"] = False
         params.default_read_target_property = None
-        self.fast_forward({})
-        with pytest.raises(AEAActException):
+        self.fast_forward({"read_stream_id": "dummy_stream_id"})
+        with pytest.raises(
+            AEAActException,
+            match="read_target_property has not been set neither in the synchronized_data nor as a default parameter",
+        ):
```

Comment:
Ok, but ideally `test_raises_a` and `test_raises_b` should be the same method with a parametrized input to eliminate code dup. The code is identical in both methods.